jsr352/core/src/main/java/org/hibernate/search/jsr352/massindexing/impl/util/CompositeIdOrder.java

```diff
@@ -113,10 +119,21 @@ private Criterion restrictLexicographically(BiFunction<String, Object, SimpleExp
 			// The last expression has whatever symbol is defined by "lastRestriction"
```

Comment:
Right, thanks for pointing this out. I pushed a commit directly to the jsr352 branch to fix this and another typo.